ui/app/models/clients/monthly.js

```diff
@@ -1,7 +1,6 @@
 import Model, { attr } from '@ember-data/model';
 export default class MonthlyModel extends Model {
   @attr('string') responseTimestamp;
-  @attr('array') byNamespace;
```

Comment:
should have removed in last PR - `byNamespace` was replaced by `byNamespaceTotalClients` and `byNamespaceNewClients`